projects/avahi/project.yaml

```diff
@@ -1,3 +1,4 @@
+base_os_version: ubuntu-24-04
```

Comment:
Can you exclude avahi? https://github.com/avahi/avahi/issues/787 should be addressed first to prevent the MSan false positives from popping up everywhere. I'll switch it in https://github.com/google/oss-fuzz/pull/14242 when it's ready.

Reply:
Done